packages/block-library/src/cover/style.native.scss

```diff
@@ -75,3 +75,30 @@
 .clearMediaButton {
 	color: $alert-red;
 }
+
+.colorPaletteWrapper {
+	min-height: 50px;
+}
+
+.paletteColorIndicator {
+	margin-top: 0;
+	margin-bottom: 0;
+	width: 32px;
+	height: 32px;
+}
+
+.paletteCustomIndicatorWrapper {
+	flex-direction: row;
+	align-items: center;
+	width: 32px;
+	height: 32px;
+}
+
+.mediaPlaceholderEmptyStateContainer {
+	height: 300;
+}
+
+.mediaPlaceholderEmptyStateTitle {
+	margin-top: 4;
+	margin-bottom: 16;
+}
```

Comment:
I think this change could be applied to all placeholders this way we could remove one prop from the component (customEmptyStateTitleStyles) what do you think?